### PR TITLE
Update readme with dependency info and troubleshooting

### PR DESCRIPTION
**Pull request by Ellie-Brakoniecki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,4 +121,24 @@ __These are the default instructions__
 
 1. When your PR is merged to main, it will be released automatically by the [Release workflow](.github/workflows/release.yml) using the version within [package.json](package.json).
    You can still merge a PR which doesn't update the version within `package.json`, but the Release workflow will fail.
-   Failure of the Release workflow means that a new release won't have been generated, the changes made to the package will not have been uploaded to PyPI and you won't be able to use the new version of the code in dashboard repos until you update the version in `package.json`. 
\ No newline at end of file
+   Failure of the Release workflow means that a new release won't have been generated, the changes made to the package will not have been uploaded to PyPI and you won't be able to use the new version of the code in dashboard repos until you update the version in `package.json`. 
+
+### Dependencies and the Release workflow:
+
+Troubleshooting
```
Comment (kateriley1) [inline README.md:128]: Could this be a heading?

File: README.md [hunk 2]
```diff
@@ -121,4 +121,24 @@ __These are the default instructions__
...
+1. run `npm audit fix` 
+1. If that doesn't work try deleting the `node_modules` folder and the `package-lock.json` file and running `npm install` again
+1. If that doesn't work take a closer look at the affected dependency by running for example `npm ls glob-parent` where `glob-parent` is the dependency that has the vulnerability warning. This will print out the dependencies in a tree-like fashion. You can try updating the top-level dependency or the dependency itself. For example if we want to update glob-parent to version we run `npm install glob-parent@6.0.2` followed by `npm update` to update all packages to their latest versions. Another alternative if `glob-parent` is being installed by another package we can overide the version by adding the following to the package.json file 
+  `"overrides": {
+    "chokidar": "3.5.3",
+    "glob-parent": "6.0.2"
+  }` 
+  and then running `npm update` (note: need to add `npm update` to the workflow file if running github actions).
+
+Node version
```
Comment (kateriley1) [inline README.md:142]: Could this be a heading?

File: README.md (hunk 1 above)
Comment (kateriley1) [inline README.md:128]:
```suggestion
#### Troubleshooting

```

File: README.md (hunk 2 above)
Comment (kateriley1) [inline README.md:142]:
```suggestion
#### Node version 

```

**Review by kateriley1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Ellie-Brakoniecki) [reply README.md:128]: Good shout!

**Pull request merged**